kubetest2-aks/deployer/up.go

```diff
@@ -56,6 +59,70 @@ type UpOptions struct {
 	K8sVersion       string `flag:"k8sVersion" desc:"--k8sVersion flag for cluster Kubernetes version"`
 }
 
+type enableCustomFeaturesPolicy struct{}
+
+type changeLocationPolicy struct{}
+
+type addCustomConfigPolicy struct {
+	customConfig string
+}
+
+func (p enableCustomFeaturesPolicy) Do(req *policy.Request) (*http.Response, error) {
+	req.Raw().Header.Add("AKSHTTPCustomFeatures", "Microsoft.ContainerService/EnableCloudControllerManager")
```

Comment:
This feature is required for AKS k8s version <1.22.0, after 1.22.0, CloudControllerManager is GA is AKS, we don't need this header. If you check the current supported AKS k8s version, they are all >1.22.0. cc @nilo19 @feiskyer 
```
➜  ~ az aks get-versions -l westus2 -o table
KubernetesVersion    Upgrades
-------------------  ------------------------
1.25.2(preview)      None available
1.24.6               1.25.2(preview)
1.24.3               1.24.6, 1.25.2(preview)
1.23.12              1.24.3, 1.24.6
1.23.8               1.23.12, 1.24.3, 1.24.6
1.22.15              1.23.8, 1.23.12
1.22.11              1.22.15, 1.23.8, 1.23.12
```

Reply:
Got it. Thanks. I thought it is some kind of toggle for custom config. For other components, like csi ones, we don't need it either, right?

Reply:
custom configure does not care about http custom feature. We need to consider this per feature, this one feature is enabled/registered/GA in the test sub, we don't need their HTTP custom feature.

Reply:
For csi ones, we don't need things like enabled-csi-xx in AKSHTTPCustomFeatures, as they are GA in AKS as well.